sem_covid/services/crawlers/scrapy_crawlers/spiders/eu_timeline_spider.py

```diff
@@ -92,10 +100,13 @@ def parse_presscorner_page(self, response):
             'href')
 
         item['press_contacts'] = list()
+        item['topics'] = list()
         press_contacts = response.xpath('//ul[@class="ecl-listing"]/li')
         for press_contact in press_contacts:
+            document_spoke_person_name = press_contact.xpath('*//div/h4/text()').get()
+            item['topics'].append(self.get_topics_by_spoke_person_name(document_spoke_person_name))
```

Comment:
if you append, you will have an array inside another array; simply assign the value instead